src/init.cpp

```diff
@@ -869,6 +880,55 @@ void ThreadImport(std::vector<fs::path> vImportFiles)
     g_is_mempool_loaded = !fRequestShutdown;
 }
 
+void PeriodicStats()
+{
```

Comment:
What would you think about something like below to ensure this method only gets called when stats are actually enabled?
```suggestion
{
    assert(gArgs.GetBoolArg("-statsenabled", DEFAULT_STATSD_ENABLE));
```